extension/httpfs/httpfs.cpp

```diff
@@ -701,6 +733,12 @@ void HTTPFileHandle::Initialize(optional_ptr<FileOpener> opener) {
 	}
 }
 
+void HTTPFileHandle::AddHeaders(HeaderMap &map) {
+	if (!http_params.bearer_token.empty()) {
+		map["authorization"] = "Bearer " + http_params.bearer_token;
```

Comment:
The client supports `set_bearer_token_auth` for that purpose btw.

Reply:
ah thanks, that's much cleaner indeed